shell/platform/darwin/ios/BUILD.gn

```diff
@@ -266,11 +179,10 @@ shared_library("ios_test_flutter") {
   ]
   deps = [
     ":flutter_framework_source",
-    ":ocmock",
+    "//build/secondary/third_party/ocmock:ocmock",
```

Comment:
`build/secondary` is redundant and is skipped elsewhere.

Reply:
i'll do this in a sec after build lands so i can test it.

Reply:
done